lms/templates/fields/field_dropdown_account.underscore

```diff
@@ -20,6 +20,13 @@
         <span class="u-field-value-readonly"></span>
     <% } else { %>
         <select name="select" id="u-field-select-<%- id %>" aria-describedby="u-field-help-message-<%- id %>">
+            <% if (preOptions) { %>
+                <optgroup label="Common Time Zones">
+                <% _.each(preOptions, function(time_zone) { %>
+                   <option value="<%- time_zone.time_zone %>"><%- time_zone.description %></option>
+                <% }); %>
+                <optgroup label="All Time Zones">
```

Comment:
I think even a little bit more generalization could be good here.  "afterPre" is really not associated with the pre-options; it's a title for the main options.  So let's model the data that way before it comes in.  Something like:

```json
[
  {
     "groupTitle": gettext("Country Time Zones"),
     "selectOptions": [...]
  },
  {
    "groupTitle": gettext("All Time Zones"),
    "selectOptions": [...]
  }
]
```

Then the template can stay stupid and generic.  (Stupid is a virtue with templates).  `groupTitle` could be null to skip adding a title line.